src/main/java/org/randywebb/wlts/beans/Household.java

```diff
@@ -18,21 +18,21 @@
  */
 public class Household extends AbstractBean {
 
-    /** Can be used for logging debugging messages */
+    /** Can be used for logging debugging messages. */
     //private static Logger log = LoggerFactory.getLogger(Household.class);
 
-    /** estimated miles per degree of latitude for Pflugerville, TX */
-    private static double milesPerLat = 68.9;
-    /** estimated miles per degree of longitude for Pflugerville, TX */
-    private static double milesPerLon = 59.7;
+    /** estimated miles per degree of latitude for Pflugerville, TX. */
```

Comment:
Hmm, this seems OK for Pflugerville, but should this be moved to an external config in case we want it to work for wards in other parts of the world?

Reply:
Would moving this to the app config file be ok? Really these values would be "good enough" for how we are using distance for anywhere in the continental US, but I'm good moving them out.

Reply:
It is OK to leave them in there if they are "good enough." I don't plan on wide distribution of this app, but I've been conditioned to think about global applications and when I saw "for Pflugerville, TX" in the comment, it raised a flag.

We currently bundle the AppConfig properties into the jar file, so they really aren't user settings. To do this "right" we'd want to consider having a second config file for user settings that we can distribute outside of the war. I don't know that we can justify doing that at the moment. Maybe the first step is to put these in the current app config properties, but add some comments in the file to delineate App Settings from User Settings so we can refactor it later if we need to expose those settings.
We could also just create a 2nd properties file user-settings.properties (or something), bundle it into the jar, and load it in AppConfig (we can load multiple properties files into a single properties object) as a cleaner way to delineate App vs User settings, and separate it out from the jar when it becomes needed / warranted.